docs/src/main/sphinx/installation/query-resiliency.rst

```diff
@@ -0,0 +1,119 @@
+======================================================
+Improve query resiliency with fault-tolerant execution
```

Comment:
Maybe 

Query processing resilience with... 

and also .. this is weird .. resilience = fault-tolerant ..